apps/insights/package.json

```diff
@@ -28,10 +28,12 @@
     "@pythnetwork/known-publishers": "workspace:*",
     "@react-hookz/web": "catalog:",
     "@solana/web3.js": "catalog:",
+    "async-cache-dedupe": "^3.0.0",
```

Comment:
We should stick to using [catalog dependencies](https://pnpm.io/catalogs) as it makes handling upgrades across the monorepo much easier if this package were to be used in other apps / packages